frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import InputLabel from '@material-ui/core/InputLabel';
+import { makeStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import uuid from "uuid";
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+
+const useStyles = makeStyles({
+  formControl: {
+    backgroundColor: 'white',
+    marginBottom: 25,
+    minWidth: 250,
+  },
+});
+
+/** 
+ * ConfigForm is a form used to create a single user configuration.
+ * It takes as input, addConfig (a function for adding a single configuration to a list of all user configurations).
+ */
+export default function ConfigForm({ addConfig }) {
+  const classes = useStyles();
+
+  const [config, setConfig] = useState({
+    id: '',
+    data: '',
+    relatedData: '',
+  });
+
+  const DEFAULT_ID = 10;
+
+  {/**TODO: Replace drop-down menu and hard-coded data options with text-input that recommends data options*/}
+  const POSSIBLE_DIMENSIONS = ["noodle", "spice", "egg", "soup", "instant noodle"];
+  const RELATED_POSSIBLE_DIMENSIONS = ["noodle", "spice", "egg", "soup", "instant noodle"];
+  const dimensions = POSSIBLE_DIMENSIONS
+  const relatedDimensions = RELATED_POSSIBLE_DIMENSIONS
```

Comment:
oh i was thinking since there are the same set of dimensions they can just be one

const POSSIBLE_DIMENSIONS = ["noodle", "spice", "egg", "soup", "instant noodle"];

dimensions = POSSIBLE_DIMENSIONS
relatedDimensions = POSSIBLE_DIMENSIONS